Add normalisation factor to Integration for RebinnedOutput #33407 (#33408)
mantidproject/mantid: docs/source/release/v6.3.0/33408_framework.rst

Algorithms
----------

Bug Fixes
#########

> - Fixed a bug in :ref:`Integration <algm-Integration>` when using it with a ``RebinnedOutput`` workspace created by :ref:`SofQWNormalisedPolygon <algm-SofQWNormalisedPolygon>` where the output was not correctly normalised.

Reviewer: ```suggestion
- Fixed a bug in :ref:`Integration <algm-Integration>` when using it with a ``RebinnedOutput`` workspace (e.g from :ref:`SofQWNormalisedPolygon <algm-SofQWNormalisedPolygon>`) where the output was not correctly normalised.
```
